lib/bundler/cli.rb

```diff
@@ -465,6 +467,7 @@ def package
     map "e" => "exec"
     def exec(*args)
       require_relative "cli/exec"
+      metrics
```

Comment:
Let's move this inside the `exec.rb` file, and record only the amount of time that it took for Bundler to run, instead of recording all the system information.